ocaml/xapi/xapi_pool_update.ml

```diff
@@ -376,3 +376,18 @@ let detach_attached_updates __context =
     ignore(Helpers.call_api_functions ~__context
       (fun rpc session_id -> Client.Pool_update.detach ~rpc ~session_id ~self ~host))
   )
+
+let pool_update_download_handler (req: Request.t) s _ =
+  debug "pool_update.pool_update_download_handler URL %s" req.Request.uri;
+  let path =
+    if String.startswith Constants.get_pool_update_download_uri req.Request.uri then
```

Comment:
Hi @thomassa 
I think we have URL like .../update/file. But the file is located as /var/update/file
So we need to do the transfer.

Also yes, as the handler is registered as /update/, the /update/ will always there. remove the check of if the URL is start with /update/